extensions/arc/src/common/workspaceUtils.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { InputValueType } from 'resource-deployment';
+import * as azExt from 'az-ext';
+import * as vscode from 'vscode';
+import { errorListingLogAnalyticsWorkspaces } from '../localizedConstants';
+
+export const _azApi = <azExt.IExtension>vscode.extensions.getExtension(azExt.extension.name)?.exports;
```

Comment:
You generally shouldn't do this outside of a class since the extension may not be activated yet when the module is loaded and this code runs. 

Just put this in the function itself so it only runs when that is called. (it'll mean calling it more than necessary, but that shouldn't be a huge deal. Feel free to cache it though if perf is a concern)